Dockerfile.worker

```diff
@@ -0,0 +1,18 @@
+FROM python:3.5-slim
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+ENV CELERY_VERSION 4.0.2
+
+COPY workers/requirements.in .
+
+RUN pip install pip-tools && \
+    pip-compile requirements.in && \
+    pip install -r requirements.txt
+
+COPY workers/ .
+
+USER user
+
+CMD ["celery", "-A", "bioinformatics_mill_workers", "-c", "1", "worker"]
```

Comment:
Although I've thought a bit about it and while it's not specific to Docker [this article](https://aws.amazon.com/ec2/spot/spot-and-science/) does describe the "Grid Architecture" pattern that the foreman/workers are going to be implementing. (Jump down to the section titled `"Grid" Architecture`, it refers to something called a "master node" which I've renamed Foreman for our purposes since we're a refinery.)